### PR TITLE
Update Azure peering TF notes

### PR DESCRIPTION
**Pull request by azhang08**
Add additional step in Azure peering TF example notes as in [official doc](https://docs.confluent.io/cloud/current/networking/peering/azure-peering.html)

### CODE REVIEW

**Review by ejagade — APPROVED** [review]

**Comment by channingdong** [comment]
Please hold on merging, we have a minor release that needs to be complete before merging more PRs, I will approve once the minor version release is finished.

**Review by channingdong — APPROVED** [review]

**Pull request merged**
